vscode.sh

```diff
@@ -128,6 +130,16 @@ if [ "${FLATPAK_ISOLATE_PIP}" -ne 0 ]; then
   fi
 fi
 
+if [ "${FLATPAK_ISOLATE_GEM}" -ne 0 ]; then
+  msg "Setting up Ruby packages"
+  GEM_USER_INSTALL="$XDG_DATA_HOME/gem/ruby/3.0.0"
```

Comment:
I think we should get the actual path instead of hardcoding one, e.g.
```suggestion
  GEM_USER_INSTALL=$(ruby <<<'puts Gem.user_dir')
```
Although invoking `ruby` interpreter is rather slow for a wrapper; ideally we should determine the path at build time.